AccountService/src/main/java/com/account/entity/Account.java

```diff
@@ -0,0 +1,50 @@
+package com.account.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name="Account")
+public class Account {
+
+	
+	@Column(name="first_name")
```

Comment:
in spring jpa = crud - u do not need @Column

Reply:
my column names in database are different thats why I have used @Column

Reply:
i did think so spring jpa convert @Column(name="last_name")
	to  lastName;